elb_automation/meaoelb/oregon_b.py

```diff
@@ -57,14 +49,24 @@
 snippets_stage.elb_config.elb_atts.connection_settings.idle_timeout = 120
 snippets_stage.elb_config.health_check.target_path = '/healthz/'
 
+# ### Snippets Prod
+snippets_stage = elb_tool.define_elb_http(
+    service_namespace='snippets-prod',
+    service_name='snippets-nodeport',
+    ssl_arn='arn:aws:acm:us-west-2:236517346949:certificate/fbe34166-ae87-43f8-b9cc-7bc9a45d904c')
+snippets_stage.elb_config.elb_atts.connection_settings.idle_timeout = 120
```

Comment:
We have `WSGI_KEEP_ALIVE=90` in snippets-prod config. Should we make those two values match?

Reply:
https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/config-idle-timeout.html


        To ensure that the load balancer is responsible for closing the connections to your instance, make sure 
        that the value you set for the HTTP keep-alive time is greater than the idle timeout setting on your 
        load balancer.